server/src/server.js

```diff
@@ -101,7 +101,7 @@ function initDatabase() {
 
 initDatabase();
 
-const csrfProtection = csrf({cookie: true});
+const csrfProtection = csrf({cookie: {httpOnly: true, secure: true, sameSite: 'lax', key: '__Host-csrf'}});
```

Comment:
We're initializing multiple csrf middleware in multiple places, so I think we need to update the configs there too or have them all share one config or middleware layer.

```
» git log -1 --format=oneline
12197ff596cf9514c24813e2aef2314481b59016 (HEAD -> master, origin/master) Merge pull request #3358 from j605/fix-shebang
» git grep -n csrf\(
server/src/pages/leave-screenshots/server.js:7:const csrfProtection = csrf({cookie: true});
server/src/pages/settings/server.js:9:app.get("/", csrf({cookie: true}), function(req, res) {
server/src/pages/shot/server.js:12:app.get("/:id/:domain", csrf({cookie: true}), function(req, res) {
server/src/pages/shotindex/server.js:11:app.get("/", csrf({cookie: true}), function(req, res) {
server/src/server.js:104:const csrfProtection = csrf({cookie: true});   # the one fixed here
```